src/Event/Events/EventFromUDB2.php

```diff
@@ -132,8 +133,8 @@ private function getCalendar(array $calendarAsArray): Calendar
 
         if ($calendarType === 'periods') {
             $dateRange = new DateRange(
-                \DateTimeImmutable::createFromFormat('Y-m-d', $calendarAsArray['periods'][0]['period'][0]['datefrom'][0]['_text']),
-                \DateTimeImmutable::createFromFormat('Y-m-d', $calendarAsArray['periods'][0]['period'][0]['dateto'][0]['_text'])
+                DateTimeFactory::fromFormat('Y-m-d', $calendarAsArray['periods'][0]['period'][0]['datefrom'][0]['_text']),
```

Comment:
Would it be a good idea to put this format in a constant?

Reply:
As discussed best to fix in dedicated ticket